xata/app/lib/settings.ts

```diff
@@ -0,0 +1,66 @@
+import type { HtmlMetaDescriptor } from '@remix-run/react'
```

Comment:
Can we get rid of this file and what it implements to keep the example focused?

Reply:
I removed everything from this file except the `LINKS` constant. That's so there's a way to perform a Write to the database from the template (so users can see it working and adjust accordingly). Let me know if you prefer to strip it out as well.